client/landing/gutenboarding/components/plans-button/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { useViewportMatch } from '@wordpress/compose';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@automattic/react-i18n';
+
+/**
+ * Internal dependencies
+ */
+import JetpackLogo from 'components/jetpack-logo'; // @TODO: extract to @automattic package
+import * as plans from 'lib/plans/constants';
+import { getPlan } from 'lib/plans';
+import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const PlansButton: React.FunctionComponent< Button.ButtonProps > = ( { ...buttonProps } ) => {
+	const { __ } = useI18n();
+	const hasPaidDomain = useSelect( ( select ) => select( ONBOARD_STORE ).hasPaidDomain() );
+	const isDesktop = useViewportMatch( 'mobile' );
```

Comment:
😕 this reads very strange… Why is `isDesktop` matching `mobile`?

Reply:
This would be the mobile-first approach that is matching our SCSS breakpoints imported from Gutenberg. The default of the second argument of `useViewportMatch` is `'>='`. So instead `isDesktop` I could say `isLargerThanMobile` 🤔 

The [current approach in Gutenboarding](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/components/header/index.tsx#L121) is to go desktop-first in JS files and mobile first in SCSS which is leaving the exact `480px` width resolution with components rendered for mobile and desktop CSS.